packages/pictograms/src/svg/default.svg

```diff
@@ -0,0 +1 @@
30.1004618 32.666054,30.1004618 Z M27.748854,20.8919618 L34.161854,24.4129618 C33.999854,22.9959618 33.163854,21.7409618 31.870854,21.0319618 C30.577854,20.3219618 29.055854,20.2839618 27.748854,20.8919618 Z M34.618854,25.5899618 C34.546854,25.5899618 34.474854,25.5729618 34.409854,25.5379618 L26.653854,21.2799618 C26.521854,21.2059618 26.436854,21.0709618 26.429854,20.9199618 C26.422854,20.7699618 26.494854,20.6259618 26.618854,20.5409618 C28.316854,19.3869618 30.488854,19.2849618 32.287854,20.2719618 C34.088854,21.2609618 35.147854,23.1409618 35.050854,25.1779618 C35.043854,25.3279618 34.959854,25.4619618 34.829854,25.5359618 C34.764854,25.5729618 34.691854,25.5899618 34.618854,25.5899618 Z M22.204454,29.0100618 L30.099454,33.3440618 L30.882454,31.9530618 L22.987454,27.6190618 L22.204454,29.0100618 Z M30.265454,34.3630618 C30.195454,34.3630618 30.123454,34.3460618 30.057454,34.3100618 L21.403454,29.5590618 C21.302454,29.5030618 21.227454,29.4090618 21.195454,29.2980618 C21.164454,29.1870618 21.178454,29.0680618 21.235454,28.9660618 L22.443454,26.8210618 C22.559454,26.6140618 22.821454,26.5400618 23.029454,26.6540618 L31.683454,31.4050618 C31.784454,31.4600618 31.859454,31.5540618 31.890454,31.6650618 C31.922454,31.7770618 31.908454,31.8960618 31.851454,31.9970618 L30.643454,34.1430618 C30.564454,34.2830618 30.417454,34.3630618 30.265454,34.3630618 Z M23.616554,35.6692618 C24.910554,36.3802618 26.433554,36.4192618 27.738554,35.8092618 L21.325554,32.2872618 C21.487554,33.7042618 22.324554,34.9602618 23.616554,35.6692618 Z M25.816554,37.1032618 C24.920554,37.1032618 24.021554,36.8792618 23.199554,36.4292618 C21.398554,35.4402618 20.339554,33.5602618 20.436554,31.5232618 C20.443554,31.3732618 20.527554,31.2382618 20.657554,31.1652618 C20.787554,31.0932618 20.946554,31.0932618 21.077554,31.1632618 L28.833554,35.4212618 C28.965554,35.4942618 29.050554,35.6302618 29.057554,35.7802618 C29.064554,35.9312618 28.992554,36.0742618 28.868554,36.1592618 C27.946554,36.7862618 26.883554,37.1032618 25.816554,37.1032618 Z M30.848254,14.8665618 C30.389254,14.8665618 29.944254,15.1045618 29.705254,15.5275618 C29.538254,15.8245618 29.497254,16.1685618 29.591254,16.4955618 C29.685254,16.8265618 29.905254,17.1005618 30.210254,17.2675618 C30.841254,17.6135618 31.638254,17.3885618 31.987254,16.7685618 C32.155254,16.4715618 32.195254,16.1295618 32.102254,15.8025618 C32.008254,15.4705618 31.788254,15.1965618 31.483254,15.0295618 C31.281254,14.9195618 31.063254,14.8665618 30.848254,14.8665618 Z M30.842254,18.2965618 C30.486254,18.2965618 30.126254,18.2095618 29.794254,18.0275618 C29.284254,17.7485618 28.917254,17.2885618 28.758254,16.7335618 C28.601254,16.1815618 28.669254,15.6025618 28.951254,15.1015618 C29.533254,14.0705618 30.856254,13.6945618 31.899254,14.2695618 C32.409254,14.5505618 32.777254,15.0095618 32.935254,15.5645618 C33.092254,16.1155618 33.024254,16.6945618 32.742254,17.1945618 C32.345254,17.8985618 31.604254,18.2965618 30.842254,18.2965618 Z"></path></svg>
```

Comment:
@chrisconnors-ibm So I got all the files moved based on my understanding after the call. Did we decide whether we wanted to keep `default.svg` for now, or change it to `bee.svg`?

Reply:
not sure. @conradennis might know the intent there.